Resources/Prototypes/Entities/Mobs/Corpses/corpses_settings.yml

```diff
@@ -0,0 +1,177 @@
+- type: randomHumanoidSettings
+  id: RandomServiceCorpse
+  randomizeName: false
+  components:
+  - type: Damageable
+    damage:
+      types:
+        Bloodloss: 49
+        Asphyxiation: 76
+        Slash: 56
+        Blunt: 19
+  - type: Inventory
+    templateId: corpse
+  - type: Loadout
+    prototypes:
+    - ClownGear
+    - MimeGear
+    - JanitorGear
+    - JanitorGear
+    - ServiceWorkerGear
+    - ServiceWorkerGear
+    - MusicianGear
+    - BotanistGear
+    - BotanistGear
+    - ChefGear
+    - ChaplainGear
+    - PassengerGear
+    - PassengerGear
+    - PassengerGear
+    - HoPGear                     # Chance 1/15
```

Comment:
Just add weightedrandomprototype support to loadouts please.

Reply:
at the moment, due to studying, I can't learn C# with all my desire - but I will do it later, since later I will have more time (maybe this year)
Thus, I have equalized the chances among themselves, and the command corpses now appears only in a specialized prototype.